src/openrct2/world/Fountain.cpp

```diff
@@ -38,12 +38,25 @@ namespace FOUNTAIN_FLAG
     const uint32_t DIRECTION = 1 << 7;
 }; // namespace FOUNTAIN_FLAG
 
-static constexpr const CoordsXY _fountainDirectionsNegative[] = {
-    { -32, 0 }, { -32, -32 }, { 0, 0 }, { -32, 0 }, { 0, 0 }, { 0, -32 }, { 0, -32 }, { -32, -32 },
+static constexpr const std::array<CoordsXY, 8> _fountainDirectionsNegative = {
```

Comment:
If these arrays were defined in terms of CoordsDirectionDelta positions would that be more readable?

Reply:
I'm not certain, because they're not a 1-1 copy of the `CoordsDirectionDelta`

Reply:
Okay thats fine.